configuration/csharp/http/README.md

```diff
@@ -0,0 +1,70 @@
+# Dapr Configuration API
+
+In this quickstart, you'll create a microservice which makes use of Dapr's Configuration API. Configuration items are key/value pairs containing configuration data such as app ids, partition keys, database names etc. The service gets configuration items from the configuration store and subscribes for configuration updates.
+
+Visit [this](https://docs.dapr.io/developing-applications/building-blocks/configuration/) link for more information about Dapr and Configuration API.
+
+> **Note:** This example leverages HTTP `requests` only.  If you are looking for the example using the Dapr Client SDK (recommended) [click here](../sdk/).
+
+This quickstart includes one service:
+
+- Dotnet service `order-processor`
+
+## Add configuration items to the config store
+
+### Prerequisite
+
+- Locally running redis instance - a redis instance is automatically created as a docker container when you run `dapr init`
+- Install Redis CLI - [Getting started with Redis | Redis](https://redis.io/docs/getting-started/). `redis-cli` is installed as part of redis setup
```

Comment:
I feel it would be a lot better for users and local mmd testing (`make validate`) if you could make this step instead do a `docker exec ...` of the command against the `dapr_redis` container that every Dapr CLI user already has.  

Could we please make this change?  then you will get the LGTM..

Reply:
Updated to use docker exec